gordon.toml.example

```diff
@@ -15,3 +15,10 @@ bar = baz
 ["foo.plugin"]
 # specific plugin config within "foo" package
 baz = bla
+
+[metrics]
```

Comment:
I agree, but since I think we're still not 100% sure this is the way we want to do metrics, I'd prefer to get the logic approved before writing a bunch of docs about it.

Reply:
Fair enough.  Let's just make sure there's a task for docs somewhere.

Reply:
I wouldn't mind doing it immediately after this is merged, but just because the config format still hasn't been approved yet, I don't want docs for it to be a part of this PR.